src/merge_bot/merge_bot.py

```diff
@@ -133,7 +133,7 @@ def message_slack(webhook_url, msg):
     if webhook_url is None:
         return
     # Cut long messages to 500 chars
-    requests.post(webhook_url, json={"text": msg[:500]})
+    requests.post(webhook_url, json={"blocks":[{"type":"section","text":{"type":"plain_text","text":msg[:500],"emoji":False}}]})
```

Comment:
The linter [doesn't like your python style](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/shiftstack_merge-bot/18/pull-ci-shiftstack-merge-bot-main-lint/1716454553118838784#1:build-log.txt%3A69):
```
-    requests.post(webhook_url, json={"blocks":[{"type":"section","text":{"type":"plain_text","text":msg[:500],"emoji":False}}]})
+    requests.post(
+        webhook_url,
+        json={
+            "blocks": [
+                {
+                    "type": "section",
+                    "text": {"type": "plain_text", "text": msg[:500], "emoji": False},
+                }
+            ]
+        },
+    ) 
```